Global.gd

```diff
@@ -226,8 +226,10 @@ func _deferred_goto_scene(scene, path, function_call = null):
 	var overworld_node = persisted_scenes[Scene.OVERWORLD]
 	if overworld_node != null:
 		if scene != Scene.OVERWORLD:
+			overworld_node.toggle_overworld_hud(false)
```

Comment:
Let's change the function name to something like `set_ui_visibility` so we can use it for any other persisted scenes.